core/src/actions/status.ts

```diff
@@ -32,6 +33,14 @@ export class PrivateStatus extends AuthenticatedAction {
     ).value;
 
     const samples = await Status.get();
+    const { leader } = await task.details();
+    const runMode = getGrouparooRunMode();
+
+    if (!leader && runMode === "cli:start") {
+      throw new Error(
+        `no leader for this Grouparoo cluster "${clusterName}" (${runMode})`
+      );
+    }
```

Comment:
This change to the private status action will ensure that we are notified if there are no workers/leaders for a given cluster